cmd/soroban-rpc/internal/daemon/daemon.go

```diff
@@ -239,10 +241,10 @@ func MustNew(cfg config.LocalConfig, endpoint string, adminEndpoint string) *Dae
 		ReadTimeout: defaultReadTimeout,
 	}
 	if adminEndpoint != "" {
-		adminMux := http.NewServeMux()
+		adminMux := supporthttp.NewMux(logger)
+		// after importing net/http/pprof, debug endpoints are implicitly registered in the default serve mux
+		adminMux.Handle("/", http.DefaultServeMux)
```

Comment:
Using explicit handlers like you had before is a bit safer since you can be sure that all the handlers are being defined here. On it's own, I'm sure that it's fine - but it might trigger undesired exposed endpoint in the future.

Reply:
Problems is there are quite a few handlers. I can do that though

Reply:
Another problem is that new handles are added with time and we will miss them.